checker/build.gradle

```diff
@@ -36,6 +36,16 @@ dependencies {
     implementation 'org.plumelib:reflection-util:1.0.3'
     implementation 'org.plumelib:plume-util:1.5.5'
 
+    // TODO: it's a bug that annotatedlib:guava requires the error_prone_annotations dependency.
```

Comment:
Can this change be put in a separate pull request and reviewed by smillst?

Reply:
This code was copied verbatim from `framework/build.gradle`, where it's already present (and has therefore been reviewed). The effect of adding it here is that guava is present in `checker` in addition to `framework`.

I suspect @smillst  wouldn't want to support this change on its own (since it would add a dependency that wasn't used). But maybe she can take a look at it here and leave a comment letting us know whether she thinks it's safe to add this dependency to `checker`. If not, I can try to rewrite the parts of this checker that rely on Guava to use some other library (or the JDK).

Reply:
This change is fine.  It's a new dependencies to checker, but it is already a dependency of framework, so it's already in checker.jar.  This should not be merged separately.